.github/workflows/appetize.yml

```diff
@@ -19,144 +19,157 @@ on:
         required: false
 
 jobs:
+  prepare:
+    name: Preparing deploy
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.matrix.outputs.result }}
+    steps:
+      - name: 📋 Download SDK information
+        run: curl --silent -o versions.json https://exp.host/--/api/v2/versions
+      - name: 🔍 Resolve SDK versions
+        uses: actions/github-script@v4
+        if: github.event.inputs.sdkVersion
+        id: sdk
+        with:
+          script: |
+            const { sdkVersions } = require('./versions.json')
+            const version = '${{ github.event.inputs.sdkVersion }}'
+            const sdk = sdkVersions[version];
+
+            if (!sdk) throw new Error(`Could not find SDK "${version}"`)
+
+            core.info(`Resolved client versions for SDK "${version}"`)
+            core.info(`  - Android: ${sdk.androidClientVersion}`)
+            core.info(`  - iOS: ${sdk.iosClientVersion}`)
+
+            for (const key of ['androidClientVersion', 'androidClientUrl', 'iosClientUrl', 'iosClientVersion']) {
```

Comment:
Using `[...].forEach` kinda caused a race condition somehow. Not sure why, but you can [see it here](https://github.com/byCedric/snack-appetize-workflow/runs/3304590518?check_suite_focus=true#step:3:27).

Reply:
this is a super weird one! it looks like the sdk object is somehow gc'd before we finish it.

Reply:
Garbage collection seems to not work great in github scripts 😄